File: many.js
"use strict";

const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const t = parseFloat(input)
const s=`${Math.sin(t)},${Math.cos(t)},${Math.tan(t)},${Math.atan(t)},${Math.atan2(t,t/2)}`;
console.log(s)

File: many.py
import math

t = float(input())

print( "%.20f,%.20f,%.20f,%.20f,%.20f" % (
  math.sin(t),
  math.cos(t),
  math.tan(t),
  math.atan(t),
  math.atan2(t,t/2)
))